### ISSUE TITLE
App steals focus when starting from my yabairc file, but not as --verbose

### ISSUE THREAD

**Issue by ognistik** [koekeishiya/yabai#2241]
I have this issue with several apps, not one in particular. Where this is the most obvious and easy to replicate is with Arc. If I am playing a YT video and try to go to a different workspace, the PiP window will pop for a second and then pull me back to the workspace which has Arc. 

Several other apps randomly don't let me switch workspace and "pull me back" to their own workspace. I am not sure, but seems to be mostly video players, or video editing apps, and it may have to do with the video of an app still playing when I switch spaces. Once I `yabai --stop-service` this issue disappears.

BTW, my yabairc has both `mouse_follows_focus` and `focus_follows_mouse` off. I do have the scripting addition and SiP partially disabled. Everything seems to be running without issues with the one exception being this focus annoying problem.

**I thought I'd try to debug this by running the service as `yabai --verbose` in terminal and to my surprise the issue is totally gone there.** Arc PiP doesn't steal the focus and it's the same with any other app.

What could be going on here? Everything runs perfectly in `--verbose` but as soon as I go back to `yabai --start-service` the problem comes back.

**Comment by afadingthought** [comment]
> This problem is very annoying, is there any update?

Unfortunately, there seems to be no fix yet.

My workaround is to minimize the window that's causing the problem. In Arc, if I want to us PiP, I have to use the [PiP Chrome Extension](https://chromewebstore.google.com/detail/picture-in-picture-extens/hkgfoiooedgoejojocmhlaklaeopbecg) to activate Pip and then minimize Arc's window. I do think the problem is Yabai and not Arc since it happens with other apps as well _(happens with [Screen Studio](https://www.screen.studio/) all the time as well)._

Super annoying.

**Comment by danilkinkin** [comment]
If I have option for fully disable to switch workspaces it's resolved the problem for me, but I can't find option in documentation

**Comment by neilord** [comment]
Hi! Have you found any solution for this one?

**Comment by ognistik** [comment]
> Hi! Have you found any solution for this one?

Nope. The problem is still there and other than minimizing the window in question I don't know what else to do yet.

**Comment by neilord** [comment]
Hi! Once again, have you found any solution for this one? Thanks!
